### ISSUE TITLE
How to add a new AI

### ISSUE THREAD

**Issue by ingesiit** [ayeo/gs1_128#11]
Hi, I tried adding a custom AI, 240 in my case.
To do so, I tried modifying the file src/SectionBuilder.php adding an item in the array $data[]. This line:
`'240' => [1, 15, 'Additional product identification assigned by the manufacturer (240) - must contains between 1-15 digits'],`
The barcode seems to generates correctly, but seems that the fnc1 is missing between the AI content nd the next AI
Do anyone can help me to understand what I'm missing? I haven't deeply investigate the code, but at a first look seems that that addiction should be enough

Kind regards

**Comment by ayeo** [comment]
I will ~add~ fix  this today for you

**Comment by ayeo** [comment]
Hi again. I am in trouble :) I've read about fcn1 and it looks to me as parenthesis serve purposes of fcn1  char here. I tested "(10)123456(240)667(400)11" using [this official tool](https://www.barcodefaq.com/barcode-properties/definitions/gs1-element-string-hri-tool) and it looks ok. Could you please provide me with more information?

PS AI (240) added ;)

**Comment by ingesiit** [comment]
Thank you for the help!
I was already using your code and all was ok; then I tried to add by myself the 240 AI and sent a test label to a new customer; he told me about the missing fcn1 after the 240 content.
Now I have tried to generate a new label with your updated code and seems ok;
I tried also to validate the generated png [here](https://www.aidecoder.com/) and was decoded correclty.
Anyway I asked more clarifications about the error to the customer; I'll revert here their reply
Kind regards
Alessandro Barisone
